.wordlist.txt

```diff
@@ -62,6 +63,7 @@ enablement
 epilog
 EPYC
 ESXi
+Ethernet
```

Comment:
```suggestion
ethernet
```

Reply:
Done in https://github.com/RadeonOpenCompute/rocm-docs-core/pull/120/commits/e862e62fb9b8669bfbd4671672c3a795b9486698